data-collection/src/main/java/refactoringml/RunSingleProject.java

```diff
@@ -37,6 +38,7 @@ public static void main(String[] args) throws Exception {
 			storeFullSourceCode = true;
 
 		} else {
+			//TODO: the length of the arguments is not inline with the Readme, there nine arguments are explained: Test files only flag is missing
```

Comment:
Good catch! We have to change the README! In previous versions, the user had to define whether s/he wanted to collect test or production data. Now, the tool always collects both, but they are identified by a `isTest` boolean in the database!

Reply:
Can you open an issue for this?

Reply:
sure